features/car/access.feature

```diff
@@ -220,3 +220,11 @@ Feature: Car - Restricted access
         Then routability should be
             | highway | toll        | bothw |
             | primary | yes         |       |
+
+    Scenario: Car - Access combinations
+        Then routability should be
+            | highway    | access     | bothw |
+            | primary    | permissive | x     |
+            | steps      | permissive | x     |
+            | footway    | permissive | x     |
+            | garbagetag | permissive | x     |
```

Comment:
Looked into this - this scenario is passing at the moment.

See #3435 for a discussion if it should pass or if the highway tag should take precedence.